www-servers/h2o/files/h2o.conf

```diff
@@ -0,0 +1,30 @@
+# see /usr/share/doc/h2o/index.html for detailed documentation
+# see h2o --help for command-line options and settings
+
+user: h2o
+pid-file: /run/h2o.pid
+access-log: /var/log/h2o/access.log
+error-log: /var/log/h2o/error.log
+
+# httpoxy mitigation (https://httpoxy.org)
+# see https://github.com/h2o/h2o/pull/996
+setenv:
+  HTTP_PROXY: ""
+
+listen: 80
+
+#listen:
+#  port: 443
+#    ssl:
+#      minimum-version: TLSv1.2
+#      certificate-file: /etc/h2o/server.crt
+#      key-file: /etc/h2o/server.key
+
+hosts:
+  "localhost:80":
+    paths:
+      "/":
+        file.dir: /var/www/localhost
```

Comment:
Hmm, maybe I'm missing something but isn't this usually `/var/www/localhost/htdocs`?